qemu/tests/balloon_check.py

```diff
@@ -223,7 +225,8 @@ def run_ballooning_test(self, expect_mem, tag):
         """
         def _memory_check_after_sub_test():
             try:
-                output = self.memory_check("after subtest", ballooned_mem)
+                output = self.memory_check("after subtest", ballooned_mem,
+                                           self.params.get("balloon_threshold", 100))
```

Comment:
This param should be passed to all the places that calls `self.memory_check`.

Reply:
Modified,thanks